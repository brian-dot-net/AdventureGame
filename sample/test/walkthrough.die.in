go east
go east
<< END OF GAME >>
