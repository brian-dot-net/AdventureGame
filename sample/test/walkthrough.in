hello
look

heeyyyyyyyy
HI
take care
look table
GET KEY
look sun
move table
move out
move table
look
look coin
get table
inv
read coin
GET COIN
read coin
inv
drop
THROW coin
read coin
take coin
look coin
exit
-- END --