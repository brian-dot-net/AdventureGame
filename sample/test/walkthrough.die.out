You are in the main room. There is a doorway to the east.
There is a table here.
> You are in the auxiliary room. There is a hole in the wall. There is a doorway to the west. There is a cliff to the east.
> You fall off a cliff. Game over.
